data/json/items/ammo/9x18.json

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "id": "9x18mm",
```

Comment:
`9x18` is almost identical to `9mm` and having the former defined in terms of the latter would help keep them in balance.